gcapi/client.py

```diff
@@ -575,30 +575,36 @@ def _upload_files(self, *, files, **kwargs):
             )
         )
 
-    def upload_cases(
+    def upload_cases(  # noqa: C901
```

Comment:
Is it possible to simplify this rather than ignoring the too-complex error?

Reply:
and this is also no longer in there... strange.